hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -113,7 +114,7 @@ public SignedStateBalancesExporter(
 
 	@Override
 	public boolean isTimeToExport(Instant now) {
-		if (periodBegin != NEVER
+		if ( periodBegin != NEVER && now.getEpochSecond() % exportPeriod <= ALLOWED_EXPORT_TIME_SKEW
```

Comment:
Leo, can you explain how this helps.

Reply:
This is a little tricky. The logic to decide whether it's time to export or not is straight forward. The complexity is the time here is not a simple clock time. It's actually consensus time which is somehow also decided by the transaction rate. If for a very special case we don't have any transaction come in for 5 seconds at the test beginning, and the moment we should export falls into that 5-second period. Then the next transaction arrives will trigger an export, and this will cause the first exported account balances file off the mark. 

This probably won't happen in real world scenario, but it may happen in test env as we always start from fresh.